pyproject.toml

```
@@ -93,3 +93,21 @@ exclude_lines = [
     'def main\(.*\):',
     'if __name__ == \(.*\):',
 ]
+
+[tool.isort]
+profile = "black"
+filter_files = true
+line_length = 130
```

```suggestion
line_length = 130
```

I am not a fan of such a long `line_length`; however, this was the default configured by `flake8` so I stuck with it.

I would rather see something akin to the `black` default of `88` characters or the default for `flake8` of `80`. The long line length makes it annoying when viewing the code in a narrow window.